### ISSUE TITLE
Select has unnecessary aria-label

### ISSUE THREAD

**Issue by khalibloo** [ant-design/ant-design#45168]
### Reproduction link

[![Edit on CodeSandbox](https://codesandbox.io/static/img/play-codesandbox.svg)](https://codesandbox.io/s/l72926)

### Steps to reproduce

1. Use a Select inside a Form.Item
2. Assign a label to the form item

### What is expected?

The form item's label should be the label of the `<input />` inside the `Select`.

### What is actually happening?

The input has an aria-label "search" which overrides the label assigned to the form item.

| Environment | Info |
| --- | --- |
| antd | 5.9.4 |
| React | 18.2.0 |
| System | Windows 10 |
| Browser | Firefox 117 |

---
The workaround is to set aria-label to undefined in the Select, but it shouldn't be necessary to do so.

**Comment by khalibloo** [comment]
This looks to be the offending code https://github.com/react-component/select/blob/d5442ba6e46d7dceeb08eed893d3ca7525e802d7/src/Selector/Input.tsx#L93
I think it should be removed, and the user should be responsible for providing a `<label />` for the Select. It would also allow the user to localize the label if needed, rather than the current hard-coded label.

If no objections, I can send a PR for this.

**Comment by liangkuku** [comment]
After carefully reviewing the code, I have found that the modification should be made at (https://github.com/ant-design/ant-design/pull/45172) .Assign the label prop of the FormItem to the aria-label prop of the child component.If there are any issues with my PR, I will make the necessary modifications promptly.

**Comment by yoyo837** [comment]
Ref: https://github.com/react-component/select/issues/973

**Comment by yoyo837** [comment]
Cause by https://github.com/react-component/select/commit/b2d241fc6a7fcea66bd2b744f017c7d3c5e75881
